.changeset/new-pets-speak.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Removing the flex-direction from the hero component and will apply the direction manually via templates and scripts
```

Comment:
Given the function of this message here is to populate our changelog, what would you say to something like this?

```suggestion
Remove explicit flex-direction from the hero component in favor of atomics.
```

Reply:
should be updated now.